From f1cdf08806caf7d71779a5fe67af797c8e3ab09e Mon Sep 17 00:00:00 2001
From: Phillip Susi <psusi@ubuntu.com>
Date: Mon, 28 Jul 2014 23:19:51 -0400
Subject: [PATCH 2/6] maint: post-release administrivia

* NEWS: Add header line for next release.
* .prev-version: Record previous version.
* cfg.mk (old_NEWS_hash): Auto-update.
---
 .prev-version | 2 +-
 NEWS          | 3 +++
 cfg.mk        | 2 +-
 3 files changed, 5 insertions(+), 2 deletions(-)

diff --git a/.prev-version b/.prev-version
index 8c50098..a3ec5a4 100644
--- a/.prev-version
+++ b/.prev-version
@@ -1 +1 @@
-3.1
+3.2
diff --git a/NEWS b/NEWS
index 49c8129..297b0a5 100644
--- a/NEWS
+++ b/NEWS
@@ -1,5 +1,8 @@
 GNU parted NEWS                                    -*- outline -*-
 
+* Noteworthy changes in release ?.? (????-??-??) [?]
+
+
 * Noteworthy changes in release 3.2 (2014-07-28) [stable]
 
 ** New Features
diff --git a/cfg.mk b/cfg.mk
index ade72d7..62a02b6 100644
--- a/cfg.mk
+++ b/cfg.mk
@@ -43,7 +43,7 @@ local-checks-to-skip = \
 # Now that we have better (check.mk) tests, make this the default.
 export VERBOSE = yes
 
-old_NEWS_hash = bd453bcf049e292a9677c094d24a29dd
+old_NEWS_hash = 85f2e93c40425e202491675d97e40166
 
 include $(srcdir)/dist-check.mk
 
-- 
1.9.3

